app/src/test/java/uk/ac/ebi/atlas/experimentpage/cellplot/CellPlotServiceTest.java

```diff
@@ -129,24 +130,40 @@ void getCellPlotParameter() {
     }
 
     @Test
-    void fetchDefaultPlotMethodWithParameterisation(){
+    void fetchDefaultPlotMethodWithParameterisation() {
         when(cellPlotDaoMock.fetchDefaultPlotMethodWithParameterisation("E-CURD-4"))
-                .thenReturn(ImmutableMap.of("umap",
+                .thenReturn(ImmutableMap.of("UMAP",
                         List.of(new Gson().fromJson("{\"n_neighbors\": 15}", JsonObject.class)),
-                        "tsne",
+                        "t-SNE",
                         List.of(new Gson().fromJson("{\"perplexity\": 20}", JsonObject.class))));
 
         assertThat(subject.fetchDefaultPlotMethodWithParameterisation("E-CURD-4")
-                .get("umap").getAsJsonObject()
+                .get("UMAP").getAsJsonObject()
                 .has("n_neighbors"));
     }
 
     @Test
-    void returnEmptyResultIfThereIsNoDefaultPlotMethodAndParameterisation(){
+    void returnEmptyResultIfThereIsNoDefaultPlotMethodAndParameterisation() {
         when(cellPlotDaoMock.fetchDefaultPlotMethodWithParameterisation("fooBar"))
                 .thenReturn(ImmutableMap.of());
 
         assertThat(subject.fetchDefaultPlotMethodWithParameterisation("fooBar"))
                 .isEmpty();
     }
+
+    @Test
+    void throwsNullPointerExceptionIfHardCodedTestMethodIsNotSameAsDBMethod() {
```

Comment:
The same comment as in line 135 (randomised strings). Thanks

Reply:
I added randomised strings and accessions for the plot type and experiment accession.

Reply:
thanks for the comment.